apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -121,6 +125,84 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
 
     const allowedScopes: string = useSelector((state: AppState) => state?.authenticationInformation?.scope);
 
+    const USER_CLAIM_UPDATE_CONNECTOR: string = "user-claim-update";
+    const ENABLE_MOBILE_VERIFICATION: string = "UserClaimUpdate.MobileNumber.EnableVerification";
+    const ENABLE_EMAIL_VERIFICATION: string = "UserClaimUpdate.Email.EnableVerification";
+    const [ isMobileVerificationEnabled, setIsMobileVerificationEnabled ] = useState<boolean>(false);
+    const [ isEmailVerificationEnabled, setIsEmailVerificationEnabled ] = useState<boolean>(false);
+
+    /**
+     * The following method gets the preference for verification on mobile and email update.
+     */
+    const getPreferences = (): void => {
+
+        const userClaimUpdateConnector: PreferenceRequest[] = [
+            {
+                "connector-name": USER_CLAIM_UPDATE_CONNECTOR,
+                properties: [
+                    ENABLE_EMAIL_VERIFICATION,
+                    ENABLE_MOBILE_VERIFICATION
+                ]
+            }
+        ];
+
+        getPreference(userClaimUpdateConnector)
```

Comment:
don't we have a `usePreference` for this API ?

Reply:
We don't have a `usePreference` for this API.

Reply:
Would it be possible to create a hook for this (`usePreference`), then we would be able to use SWR as well improviing the performance

Reply:
will look into it.

Reply:
retrieving connector config preferences is actually a POST request, AFAIK we do not create hooks for POST request.